about/release-notes/index.md

```diff
@@ -23,10 +23,7 @@ click `Watch`, select `Custom` and then check `Releases`.
 
 </Highlight>
 
-This page contains the release notes from TimescaleDB [v2.10.0](#timescaledb2100-on-2023-02-21) 
-
-to [v2.15.1](#timescaledb2151-on-2024-05-28). For release notes for older versions, see the
-[past releases section][past-relnotes].
+This page contains the release notes from TimescaleDB [v2.10.0](#timescaledb2100-on-2023-02-21) to [v2.15.1](#timescaledb2151-on-2024-05-28). 
```

Comment:
Local links don't work when the docs are published. It has to be a full link like [about-hypertables]: /use-timescale/:currentVersion:/hypertables/.